src/services/grocerylist/GroceryListService.js

```diff
@@ -1,67 +1,273 @@
 import { BaseService } from "../base";
+import { logError } from "../../logger/Logger";
+import { GroceryList, FoodItem, Container } from "../../models";
 
 export class GroceryListService extends BaseService{
 
+    GROCERY_LIST_COLLECTION = 'GROCERY_LIST_COLLECTION'
+
     /**
-     * Creates a new grocery list
-     * @param {string} name 
-     * @param {string} ownerId 
+     * Creates a new grocery list and returns the grocery list object with an id
+     * @param {string} name name of the grocery list
+     * @param {string} ownerId the household/person the container belongs to
+     * @returns {GroceryList} the groceryList object that was created
+     * @throws error if grocery list was not able to be made in DB
      */
-    createGroceryList(name, ownerId){
-        throw new Error('Not Implemented')
+    async createGroceryList(name, ownerId = null){
```

Comment:
So why would we ever have an ownerId of null? Could explain your thought process on this?

Reply:
This is preparation for doing personal/private grocery list. If the grocery list is public to the household then there is no specific id for an owner, therefore it'll be null. If it ends up being a personal grocery list, there needs to be an owner id to do security checks, especially if we want to implement the idea of personA making a grocery list that is visible to personB but personB cannot add or remove items from a grocery list. Thoughts?